scalableminds/webknossos
# Refetch search query after refresh or rescan datasets was used

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- search for something
- add a dataset which matches the query
- click the rescan button --> DS should appear in the list
